spec/features/user_logs_in_spec.rb

```diff
@@ -25,33 +25,57 @@
       visit '/login'
       fill_in('session[email]', with: 'example@example.com')
       click_link_or_button 'Login'
-      within("#errors") do
+      within(".flash") do
         expect(page).to have_content("Invalid Login")
       end
     end
 
-  end
+    scenario 'logging in does not clear the cart' do
+      user = User.create(full_name: 'dg',
+                  email: 'example@example.com',
+                  password: 'password')
+      create(:item)
+      visit '/menu'
+      within("#items") do
+        click_link_or_button 'Add To Cart'
+      end
+      visit '/cart'
+      expect(page).to have_content("Sushi")
+      expect(page).to_not have_content("Onigiri")
+      expect(page).to have_content("1")
+      visit '/login'
+      fill_in('session[email]', with: user.email)
+      fill_in('session[password]', with: user.password)
+      click_link_or_button 'Login'
+      expect(page).to have_content("You are logged in")
+      visit '/cart'
+      expect(page).to have_content("Sushi")
+      expect(page).to_not have_content("Onigiri")
+      expect(page).to have_content("1")
+      end
+    end
```

Comment:
`end` at 56, 4 is not aligned with `context 'with invalid log in credentials' do` at 5, 2